README.md

```diff
@@ -21,6 +24,20 @@ Verify that your application uses DXVK instead of wined3d by checking for the pr
 
 In order to remove DXVK from a prefix, remove the DLLs and DLL overrides, and run `wineboot -u` to restore the original DLL files.
 
+### External tools
+
+On Linux there exists a wide ecosystem of programs to automate setting up DXVK or even let it auto-update. To name a few tools
```

Comment:
> On Linux there exists a wide ecosystem of programs to automate setting up DXVK or even let it auto-update

Might be better to reword this a little bit so it doesn't sound like these tools are just to manage DXVK. They are really wine managers.